client/src/__tests__/app.test.jsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter as Router } from 'react-router-dom';
+// import userEvent from '@testing-library/user-event';
+
+import App from '../App';
+
+test ('App renders correctly', () => {
+
+    //reders the app component
+    render(
+        <Router>
+            <App />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!